Gruntfile.js

```diff
@@ -511,7 +517,8 @@ module.exports = function (grunt) {
 
         grunt.task.run([
             'bower',
-            'browserify:objectHistory',
+            'browserify:externalModules',
+            'browserify:keysim',
```

Comment:
`objectHistory` is in `externalModules` now.
`keysim` is not part of that because it is only used for testing.